data_model/master/clusters/GeneralCommissioningCluster.xml

```diff
@@ -65,7 +65,7 @@ Davis, CA 95616, USA
   </clusterIds>
   <classification hierarchy="base" role="utility" picsCode="CGEN" scope="Node"/>
   <features>
-    <feature bit="0" code="TC" name="Terms &amp; Conditions" summary="Supports Terms &amp; Conditions acknowledgement">
+    <feature bit="0" code="TC" name="Terms" summary="Supports Terms &amp; Conditions acknowledgement">
```

Comment:
This needs to be addressed in the spec, if I remember right there is already an issue mentioning this

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/10078